resources/params.mjs

```diff
@@ -0,0 +1,42 @@
+
+class Params {
+    viewport = {
+        width: 1366,
```

Comment:
Can we still with 800x600 for now?

Reply:
+1 to decoupling the size change into another issue/PR, then we can get the refactor landed now